RHEL/7/input/profiles/ospp-rhel7-server.xml

```diff
@@ -0,0 +1,135 @@
+<Profile id="ospp-rhel7-server" extends="common">
+<title override="true">Protection Profile for General Purpose Operating Systems Version 4.0 - Red Hat Enterprise Linux 7 Server</title>
+<description override="true">This profile is being developed under the National Information Assurance Partnership - The scope of this Protection Profile (PP) is to describe the security functionality of operating systems in terms of Common Criteria and to define functional and assurance requirements for such products. An operating system is software that manages computer hardware and software resources, and provides common services for application programs. The hardware it manages may be physical or virtual.</description>
+
+<!-- Security Management (FMT): FMT_MOF_EXT.1 -->
+<!-- https://www.niap-ccevs.org/pp/PP_OS_v4.0/ -->
+<!-- https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Security_Guide/ -->
```

Comment:
What is the connection between the RHEL7 Security Guide off docs.redhat.com to this profile?